lib/core/theme/font_weight_helper.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+
+class FontWeightHelper {
+  static const FontWeight thin = FontWeight.w100;
+  static const FontWeight extraLight = FontWeight.w200;
+  static const FontWeight light = FontWeight.w300;
+  static const FontWeight regular = FontWeight.w400;
+  static const FontWeight medium = FontWeight.w500;
+  static const FontWeight semiBold = FontWeight.w600;
+  static const FontWeight bold = FontWeight.w700;
+  static const FontWeight extraBold = FontWeight.w800;
+}
```

Comment:
this FontWeightHelper class is unnecessary, as the FontWeight constants already exist in Flutter's TextStyle system. Flutter provides these constants directly via the FontWeight class, so creating an additional class to wrap them doesn’t add value and increases redundancy.

You can directly use FontWeight.w100, FontWeight.w200, etc., in your code without needing this helper class.